src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -156,10 +156,22 @@ public void addParticipant(Participant participant) throws AlfredException {
      *
      * @param id
      * @param participant
-     * @return boolean
      */
-    public boolean updateParticipant(Id id, Participant participant) {
-        return this.participantList.update(id, participant);
+    public void updateParticipant(Id id, Participant participant) throws AlfredException {
+        // TODO: Customize error message.
+        try {
+            // Update the participant in the team list as well
+            Team targetTeam = this.getTeamByParticipantId(id);
+            boolean isSuccessful = targetTeam.updateParticipant(participant);
+            if (!isSuccessful) {
+                logger.warning("The participant is not in the team provided");
+                return;
```

Comment:
Think you should throw exception with message being something like participant is not in team provided

Reply:
Yep this is temp, fix coming